jdisc_http_service/src/main/resources/configdefinitions/jdisc.http.connector.def

```diff
@@ -78,5 +78,11 @@ ssl.sslKeyManagerFactoryAlgorithm   string   default="SunX509"
 # The SSL protocol passed to SSLContext.getInstance()
 ssl.protocol                        string   default="TLS"
 
-# Whether connector requires client authentication. See SSLEngine.getNeedClientAuth() for details.
-ssl.needClientAuth                  bool     default=false
+# Client authentication mode. See SSLEngine.getNeedClientAuth()/getWantClientAuth() for details.
```

Comment:
Let us hope no one uses this one. The deploy will break.....

Reply:
I doubt someone discovered it after 1 day and no documentation :)